feature/cfm/CFM-1.1 Connectivity fault management/README.md

```diff
@@ -0,0 +1,747 @@
+            "profile-name": "cfm_delay_Bundle-Ether43_1600",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_delay_Bundle-Ether43_1600"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "DMM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_delay_Bundle-Ether5_4010",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_delay_Bundle-Ether5_4010"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "DMM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_delay_Bundle-Ether5_4050",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_delay_Bundle-Ether5_4050"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "SLM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_loss_Bundle-Ether43_1100",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_loss_Bundle-Ether43_1100"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "SLM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_loss_Bundle-Ether43_1600",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_loss_Bundle-Ether43_1600"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "SLM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_loss_Bundle-Ether5_4010",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_loss_Bundle-Ether5_4010"
+        },
+        {
+          "config": {
+            "burst-interval": 10000,
+            "intervals-archived": 5,
+            "measurement-interval": 1,
+            "measurement-type": "SLM",
+            "packet-per-burst": 100,
+            "packets-per-meaurement-period": 60,
+            "profile-name": "cfm_loss_Bundle-Ether5_4050",
+            "repetition-period": 1
+          },
+          "profile-name": "cfm_loss_Bundle-Ether5_4050"
+        }
+      ]
+    }
+  },
+}
```

Comment:
there is a typo/extra curly brace somewhere in the json?  You may have better luck generating the JSON with a ygot script.  See this[ example ygot script](https://github.com/openconfig/featureprofiles/blob/2bf18437e4173fd07457f8862a67c454e3506308/doc/canonical_oc_generation_example/main.go).